vermouth/graph_utils.py

```diff
@@ -197,7 +230,7 @@ def isomorphism(reference, residue):
                 if len(H_names[res_H_name]) != 1:
```

Comment:
The case where this case is `False` is not tested.

Reply:
I'll leave this as is for now. That part of the code (and the MCS) is due for refactor anyway.